cli/install/install.go

```diff
@@ -404,6 +440,8 @@ func copyBuildedTT(binDir, path, version string, installCtx InstallCtx,
 // installTt installs selected version of tt.
 func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 	versions, err := getTTVersions(installCtx.Local, distfiles)
+	versionFound := false
```

Comment:
There is no need to obtain program versions if a commit was requested. I think, we must check the input "version" format and decide what must be obtained.

Reply:
fixed